### ISSUE TITLE
Fatal Exception installing GTNH

### ISSUE THREAD

**Issue by paradust7** [FTBTeam/FTB-App#995]
### What Operating System

Windows 10

### Debug Code

FTB-DBGUHULIWAROD

### Describe the bug

GTNH and many other packs will not install

### Steps to reproduce

1. Try to install GTNH (GT New Horizons) from Curseforge.

The relevant log entries:

```
[11:16:45.842] [ForkJoinPool-1-worker-2/INFO] [net.creeperhost.creeperlauncher.data.modpack.ModpackManifest]: Querying Modpack version manifest: https://api.modpacks.ch/public/curseforge/252507
[11:16:46.596] [ForkJoinPool-1-worker-2/INFO] [net.creeperhost.creeperlauncher.data.modpack.ModpackVersionManifest]: Querying Modpack version manifest: https://api.modpacks.ch/public/curseforge/252507/4735741
[11:16:47.225] [ForkJoinPool-1-worker-2/ERROR] [net.creeperhost.creeperlauncher.data.modpack.ModpackVersionManifest]: Failed to request manifest got: No valid curseforge manifest could be found.
```

`https://api.modpacks.ch/public/curseforge/252507/4735741` contains the string:

    "message": "No valid curseforge manifest could be found.",

So perhaps the actual problem is not in FTB app, but in whatever generates these files.

### Expected behaviour

Successful install

**Comment by paradust7** [comment]
ATM8 is also affected by this problem now. Existing ATM8 installs are unable to launch, even when they are not updated, because the manifest is updated before launching.

The log entries for ATM8:
```
Querying Modpack version manifest: https://api.modpacks.ch/public/curseforge/520914
Querying Modpack version manifest: https://api.modpacks.ch/public/curseforge/520914/4787465
Failed to request manifest got: No valid curseforge manifest could be found.
```

**Comment by jmeggitt** [comment]
I am also encountering this issue. It seems likely that something changed in the manifest format breaking all packs with the older manifest version.

**Comment by Goldsaif** [comment]
I've tried installing throguh curseforge and the game still works through there, I also went digging through the logs and i came across this 

[13:02:48] [ForkJoinPool-1-worker-2/ERROR] [net.creeperhost.creeperlauncher.data.modpack.ModpackVersionManifest]: Failed to request manifest got: No valid curseforge manifest could be found.
[13:02:48] [ForkJoinPool-1-worker-2/ERROR] [net.creeperhost.creeperlauncher.api.handlers.instances.InstallInstanceHandler]: Error preparing Modpack installation.
java.lang.NullPointerException: Cannot invoke "net.creeperhost.creeperlauncher.data.modpack.ModpackVersionManifest.getId()" because "versionManifest" is null

it seems to crash right after then.
